src/analytics/tangle/mod.rs

```diff
@@ -6,13 +6,17 @@
 use crate::types::{ledger::BlockMetadata, stardust::block::Block, tangle::MilestoneIndex};
 
 mod block_activity;
+mod milestone_size;
 
-pub use self::block_activity::{BlockActivity, BlockActivityAnalytics};
+pub use self::{
+    block_activity::{BlockActivity, BlockActivityAnalytics},
+    milestone_size::{MilestoneSizeAnalytics, MilestoneSizeMeasurement},
+};
 
 #[allow(missing_docs)]
 pub trait BlockAnalytics {
     type Measurement;
     fn begin_milestone(&mut self, index: MilestoneIndex);
-    fn handle_block(&mut self, block: &Block, block_metadata: &BlockMetadata);
+    fn handle_block(&mut self, block: &Block, raw_block: &[u8], block_metadata: &BlockMetadata);
```

Comment:
Use `Raw<Block>` here.

Reply:
We can't. The methods take ownership so we can't use it as a reference.

Reply:
Indeed we'ld have to move the raw bytes and I kinda doubt that's what we would want to impose for that interface.